src/main/java/woohakdong/server/domain/member/Member.java

```diff
@@ -0,0 +1,39 @@
+package woohakdong.server.domain.member;
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import woohakdong.server.domain.school.School;
+
+@Entity
+@Getter
+@Setter
+public class Member {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long memberId;
+    private String memberProvideId;
+    private String memberName;
+    private String memberEmail;
+    private String memberPhoneNumber;
+    private String memberMajor;
+    private String memberStudentNumber;
+    private MemberGender memberGender;
```

Comment:
```suggestion
    @Enumerated(EnumType.STRING)
    private MemberGender memberGender;
```

이렇게 수정해주시면 좋을 것 같습니다. 기본 설정이 ORDINAL이라서, 이는 ENUM의 종류가 바뀔 때 예상하지 못한 결과를 가져올 수 있습니다.